src/main/java/reactor/core/publisher/FluxCreate.java

```diff
@@ -114,23 +115,23 @@ public void subscribe(Subscriber<? super T> t) {
 
 		volatile boolean done;
 
-		public SerializedSink(BaseSink<T> sink) {
+		SerializedSink(BaseSink<T> sink) {
 			this.sink = sink;
-			this.queue = QueueSupplier.<T>unbounded(QueueSupplier.XS_BUFFER_SIZE).get();
+			this.queue = QueueSupplier.<T>unbounded(16).get();
```

Comment:
We can address that separately with configuration override via Context later. Thee create doesn't open the queue argument and if there is such a race that 16 threads are trying to produce then create itself is missused.